dxr/static/js/image_preview.js

```diff
@@ -0,0 +1,19 @@
+$(function() {
```

Comment:
Why not merge this into dxr.js and save a request? (Someday we'll minify our JS and this won't matter.)